docs/intro.rst

```diff
@@ -74,13 +46,43 @@ needed and organize them in a local :doc:`datastore <dev/datastore>`.
 .. _etl-process:
 
 ---------------------------------------------------------------------------------------
-The Data Warehouse Design
+The ETL Pipeline
```

Comment:
I'm tempted to move this information to the development section of the docs. Do users actually care about the raw data archives, data warehouse and data validation?

I'm thinking we could move the [The Data Warehouse Design and Data Validation sections](https://catalystcoop-pudl--2874.org.readthedocs.build/en/2874/intro.html#the-data-warehouse-design) to the [Data and ETL Design Guidelines page](https://catalystcoop-pudl--2874.org.readthedocs.build/en/2874/dev/data_guidelines.html)?

Reply:
I'm a little hesitant to have an ETL and Data Warehouse section because they cover similar topics. I think it's easier to think about our data processing just in terms of the raw, core, output layers as opposed to ETL steps.

Reply:
Ok! As long as the concept of the Data Warehouse doesn't get lost in the code processing description I think that's fine. My concern with wanting to pull out the Data Warehouse section was similar to your comment above about people being primarily concerned with Data Access and therefore wanting to be able to jump strait to a Data Warehouse page / section might be nice, but I think depending on how we structure the rest of the docs this might not be an issue.